indymeet/settings/dev.py

```diff
@@ -11,6 +11,13 @@
         }
     }
 
+print('----------------------------------')
+print('----------------------------------')
+print('DEV')
+print('----------------------------------')
+print('----------------------------------')
+
+
```

Comment:
@dawnwages Was this intentionally left in?

Reply:
Yea it was but I can take them out if you think they're ugly.

Reply:
Nope, you are good. Keep it in, i just thought you might have forgotten. :+1: